app/views/qsets/show.html.erb

```diff
@@ -14,83 +14,25 @@
   </div>
 
   <div class="row question_list">
```

Comment:
We only need one `<div class="row question_list">`, so you can either remove this one or the one in the `_question_list_filter`.